circle.yml

```diff
@@ -7,7 +7,7 @@ dependencies:
   override:
     - cp server/config/local.json-dist server/config/local.json
     - npm i --production
-    - node_modules/.bin/grunt build
+    - CONFIG_FILES=server/config/local.json,server/config/production.json,tests/ci/config_circleci.json node_modules/.bin/grunt build
```

Comment:
Adding `CONFIG_FILES` here ensures the same config is used to build as for running the server, which has the side effect that only the locales needed for testing are built.